low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -25,9 +25,7 @@
 import traceback
 import os
 import syslog
-import subprocess
-# Add the top level directories
-sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level')
+from cortx.utils.process import SimpleProcess
```

Comment:
File changes for sspl_setup are from Satish's PR #279 , But I need those changes to remove use of Utility file from sspl,
So please do review on changes made in InitCmd.proces(), do not review changes any other changes as those will be reviewed in Satish's PR.